webpack.config.js

```diff
@@ -1,52 +1,79 @@
-// webpack.config.js
 var Encore = require('@symfony/webpack-encore');
 
-const CopyWebpackPlugin = require('copy-webpack-plugin');
+// Manually configure the runtime environment if not already configured yet by the "encore" command.
+// It's useful when you use tools that rely on webpack.config.js file.
+if (!Encore.isRuntimeEnvironmentConfigured()) {
+    Encore.configureRuntimeEnvironment(process.env.NODE_ENV || 'dev');
+}
 
 Encore
-    // the project directory where all compiled assets will be stored
-    .setOutputPath('web/build/')
-
-    // the public path used by the web server to access the previous directory
-    .setPublicPath(Encore.isProduction() ? '/pelagos-symfony/build' : '/build')
+    // directory where compiled assets will be stored
+    .setOutputPath('public/build/')
+    // public path used by the web server to access the output path
+    .setPublicPath('/build')
```

Comment:
Will this work as expected from the pelagos-symfony folder without adding it here?

Reply:
Yes, and test?

Reply:
https://griidc-dev.tamucc.edu/pelagos-symfony/data-discovery  Confirmed on griidc-dev.tamucc.edu it seems to need the pelagos-symfony directory specified to work. I'll double check all the .env variables though to make sure I didn't leave anything out.

BTW, I seem to have broke drupal on griidc-dev, which I'll fix later today. Evidently updating all drupal modules wasn't good after all.

Reply:
This file needed some attention. Should be fixed now.